AfroSellAndroidClient/app/src/main/java/com/erihackton/shopping/source/remote/AfroSellRemoteDataSource.java

```diff
@@ -21,7 +23,7 @@
 
 public class AfroSellRemoteDataSource implements DataSource {
     public static final String TAG = "AfroSellRemote";
-    public static final String baseUrl = "http://192.168.43.114:8000";
+    public static final String baseUrl = "http://192.168.43.114:3300";
```

Comment:
Aelaf, why do we need to change this port I didn't see any update in the server side actually.

Reply:
initially i was using my node.js server as a mock server, which was at port 8000 as to test my android app... now i'm using product spring server it's at 3300... but i'm continously changing the ip address..  when i'm using a internet cafe router that has dynamic ip and my phone hotspot